constants/constants.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  RUNNING_APP_MSG: 'App is successfully running on',
+  FAILED_RUNNING_APP_MSG: 'Error occurred! Server cannot start ',
```

Comment:
Suggestion: refactor it like:
{
  responseMessages: {
    authenticateUser: {
      success: '...',
      failure: '...'
    },
  }
}

Reply:
Done